components/hab/plan.sh

```diff
@@ -23,6 +23,7 @@ pkg_bin_dirs=(bin)
 bin=$_pkg_distname
 
 _common_prepare() {
+  attach
```

Comment:
I'm not very familiar with this. Can you explain why it wasn't necessary before? Or was this always missing?

Reply:
I think this might be mistakenly added?

Reply:
Verily 
https://github.com/habitat-sh/habitat/pull/5491#issuecomment-414700519